### ISSUE TITLE
Exported functions become "stale" on hot reload

### ISSUE THREAD

**Issue by enyo** [sveltejs/vite-plugin-svelte#290]

**Comment by dominikg** [comment]
i suppose this is one for svelte-hmr. cc @rixo

**Comment by rixo** [comment]
Yes, I confirm this is independent from the `context=module` issue, and that this is the responsibility of `svelte-hmr`. This one should be easier, since it pertains to component instances, where `svelte-hmr` is really acting.

Should we transfer the issue to `svelte-hmr` or create another one there?

**Comment by bluwy** [comment]
I don't have permissions to transfer over to svelte-hmr, unless someone does 👀 Else I'm fine recreating an issue there too.
